app/views/foods/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Search Results for <%= params[:q] %> </h1>
+
+
+<h3>Total search results: <%= @search_count %></h3>
+
+<% @foods[0..9].each do |food| %>
```

Comment:
As I mentioned in another comment, limiting the number of foods returned seems more like a Facade responsibility.